src/Results.svelte

```diff
@@ -17,203 +22,207 @@
-      </p>
-      <p>
-        Anträge auf Steuerstundungen und/oder auf Anpassungen von Vorauszahlungen, die nach dem 31.12.2020 eingehen oder nur Zeiträume nach dem 31.12.2020 betreffen, sind besonders zu begründen.
-      </p>
-      <p>
-      
-        Antragsformular zur Stundung der Einkommensteuer, Körperschaftsteuer und Umsatzsteuer sowie zur Herabsetzung der Vorauszahlungen zur Einkommen- und Körperschaftsteuer und des Gewerbesteuer-Messbetrages:
-      </p>
-    </div>
-    <div class="info-link-wrapper">
+    <ul class="nav nav-pills sticky-top">
+      <!--<li class="nav-item">
+          <a class="nav-link" on:click={() => selectedTab = UEBERSICHT}
+             class:active={selectedTab === UEBERSICHT} href="{'#' + UEBERSICHT}">{UEBERSICHT}</a>
+        </li>-->
       {#if selection.state}
-        <a target="_blank" class="info-link" href={steuerstundungen[selection.state]}>Antragsformular</a>
-        <a target="_blank" class="info-link" href={finanzaemter[selection.state]}>Finanzamt</a>
+        {#await data$ then data}
+          {#each data as cluster}
+            <li class="nav-item">
+              <a class="nav-link" on:click={() => selectedTab = cluster.name}
+                 class:active="{selectedTab === cluster.name}"
+                 href="{'#' + cluster.name}">{cluster.name}</a>
+            </li>
+          {/each}
+        {/await}
+      {/if}
+      <li class="nav-item">
+        <a class="nav-link" on:click={() => selectedTab = STEUERSTUNDUNG}
+           class:active={selectedTab === STEUERSTUNDUNG} href="{'#' + STEUERSTUNDUNG}">{STEUERSTUNDUNG}</a>
+      </li>
+      {#if selection.employees !== 0}
+        <li class="nav-item">
+          <a class="nav-link" on:click={() => selectedTab = KURZARBEIT}
+             class:active={selectedTab === KURZARBEIT} href="{'#' + KURZARBEIT}">{KURZARBEIT}</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" on:click={() => selectedTab = SOZIALBEITRAEGE}
+             class:active={selectedTab === SOZIALBEITRAEGE} href="{'#' + SOZIALBEITRAEGE}">{SOZIALBEITRAEGE}</a>
+        </li>
+      {/if}
+    </ul>
+  </div>
+  <div id="tab-content" class="mb-3">
+    {#await data$ then data}
+      {#each data as cluster}
+        {#if selectedTab === cluster.name}
+          <Table offers={cluster.offers}/>
+          {#if help[cluster.name]}
+            <div class="info-text">
+              {@html help[cluster.name].text}<br>
+              <a target="_blank" href={help[cluster.name] ? help[cluster.name].link : "https://wir-bleiben-liqui.de"}>
+                Sie wollen mehr wissen? Direkt zu unserem Blog
+              </a>
+            </div>
+          {/if}
+        {/if}
+      {/each}
+    {/await}
+    <div class="info-text">
+      {#if selectedTab === STEUERSTUNDUNG}
+        <p>
+          Ihr Bundesland <b>{$bundeslaender.find(land => land.id == selection.state).name || ''}</b> bietet
```

Comment:
So the whole Steuerstundung tab should not be shown in case state is not set? Feel free to implement a fix as you have the best knowledge in terms of what content should be displayed in what cases